docs/receipt-verifier.md

```diff
@@ -0,0 +1,118 @@
+sudo docker run -p 6379:6379 -d redis
+npm start
+```
+
+## Update your Web Monetization meta tag
+
+Typically, a Web Monetization meta tag looks something like this:
+
+`<meta name=“monetization" content="$wallet.example.com/alice">`
+
+For queries from the WM sender to be proxied by the receipt verifier service to your payment pointer:
+
+1. URI-encode your payment pointer. See [encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) on MDN Web Docs if you need help.
+2. Add your pointer to the path of the receipt verifier service's URL.
+3. Update your meta tag's `content` to be this new value.
+
+For example, if your payment pointer is `$wallet.example.com/alice` and your receipt verifier service’s URL is `https://receipt-verifier.com`, then you’ll set your meta tag’s `content` to either of the following:
+
+* `https://receipt-verifier.com/%24wallet.example.com%2Falice`
+* `$receipt-verifier.com/%24wallet.example.com%2Falice`
+
+
+## Web Monetization revshare generator
+
+If you used the [Web Monetization revshare generator](https://webmonetization.org/prob-revshare) to create a meta tag for [probabilistic revenue sharing](https://webmonetization.org/docs/probabilistic-rev-sharing/), follow the same instructions as above: URI-encode your revshare payment pointer and put it in the path of the receipt verifier service’s URL.
+
+For example, if your meta tag looks like this:
+
+`<meta name="monetization" content="$webmonetization.org/api/revshare/pay/W1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0">`
+
+And your receipt verifier service’s URL is `https://receipt-verifier.com`, then you’ll set your meta tag’s `content` to either of the following:
+
+* `https://receipt-verifier.com/%24webmonetization.org%2Fapi%2Frevshare%2Fpay%2FW1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0`
+* `$receipt-verifier.com/ %24webmonetization.org%2Fapi%2Frevshare%2Fpay%2FW1siJHdhbGxldC5leGFtcGxlL2FsaWNlIiwxMCwiQWxpY2UiXSxbIiR3YWxsZXQuZXhhbXBsZS9ib2IiLDEwLCJCb2IiXV0`
+
+## Set up a monetizationprogress event listener
+
+When a WM sender receives a receipt, the sender notifies the user agent of the successful payment by submitting a `monetizationprogress` event containing a receipt.
+
+Add the following client-side code to your website to listen for a `monetizationprogress` event containing a `receipt`. When a receipt exists, your website will verify the receipt in the event listener to confirm the payment.
+
+```html
+<head>
+  <!-- This should be set to .... -->
+  <meta name="monetization" content="https://receipt-verifier.com/%24wallet.example.com%2Falice">
+  <script>
+    if (document.monetization) {
+        document.monetization.addEventListener('monetizationprogress', event => {
+            // A payment has been received
+
+            // Connect to your site’s backend to validate the payment
```

Comment:
let's add a note like "this _does not_ connect directly to the receipt verifier" to make it even clearer. I feel like this is something people could get tripped up on